components/common/actionButton/deletePopover.tsx

```
@@ -58,9 +63,11 @@
           </Row>
         </div>
       }
-    >
-      {children}
-    </Popover>
+         >
+       <div onClick={() => setOpen(true)}>
+         {children}
+       </div>
```

ESLint

[error] Replace `·······<div·onClick={()·=>·setOpen(true)}>⏎·········{children}⏎·······` with `······<div·onClick={()·=>·setOpen(true)}>{children}`

(prettier/prettier)